app/src/main/java/com/simplecity/amp_library/model/Song.java

```diff
@@ -1,17 +1,5 @@
 package com.simplecity.amp_library.model;
 
-import android.content.ContentUris;
```

Comment:
That's a lot of changes to remove a single line of code! If we're committing formatting changes, they need to be consistent with the rest of the app. (Need to revert import rearrangement)